unlock-app/src/stories/creator/KeyList.stories.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Provider } from 'react-redux'
+import KeyList from '../../components/creator/lock/KeyList'
+import createUnlockStore from '../../createUnlockStore'
+
+const sampleLocks = {
+  '0': {
+    address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e0',
+  },
+  '10': {
+    address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e10',
+  },
+  '50': {
+    address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e50',
+  },
+  '100': {
+    address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e100',
+  },
+  '200': {
+    address: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e200',
+  },
+}
+
+const addSampleKeys = () => {
+  const keys = {}
+  Object.keys(sampleLocks).forEach((numberStr, i) => {
+    const numberOfKeys = parseInt(numberStr, 10)
+    Array(numberOfKeys)
+      .fill()
+      .forEach((_, j) => {
+        keys[`${numberStr} ${j}`] = {
+          id: `${numberStr} ${j}`,
+          transaction: `0x23749328748932748932473298473289473298${i}${j}`,
```

Comment:
```suggestion
          transaction: `0x${i}${j}23749328748932748932473298473289473298`,
```

The end of the key is cut off so we can't actually see that the pagination works when clicking through pages